src/vm/threadstatics.cpp

```diff
@@ -500,14 +500,20 @@ void    ThreadLocalModule::AllocateDynamicClass(MethodTable *pMT)
     // We need this check because maybe a class had a cctor but no statics
     if (dwStaticBytes > 0 || dwNumHandleStatics > 0)
     {
```

Comment:
Do these get freed up when the code is unloaded? (I am not able to see where it happens.)

Reply:
Hmm, I was convinced that the whole ThreadLocalModule goes away when the related module is unloaded, but stepping through the code, I can see it doesn't happen. I need to investigate that.

Reply:
Fixed by an additional commit